dev-lisp/c2ffi/c2ffi-11.0.0.20201113.ebuild

```diff
@@ -16,8 +16,8 @@ SLOT="0"
 KEYWORDS="~amd64"
 IUSE=""
 
-DEPEND="sys-devel/clang:10=
-	sys-devel/llvm:10=
+DEPEND="sys-devel/clang:11=
+	sys-devel/llvm:11=
```

Comment:
Have we talked about this before, but I can have llvm-10 clang-10 and -11 installed, but how does this ebuild force -11 to be used? You probably need some checks from *llvm.eclass*.

Reply:
Or does upstream handle this with their build system properly?

Reply:
AFAICT, the build system will look for llvm-11, not llvm-10. So if llvm-11 is not installed, it will not build.